distributions/openhab/src/main/resources/conf/services/runtime.cfg

```diff
@@ -77,3 +77,15 @@ org.openhab.ephemeris:dayset-school=[MONDAY,TUESDAY,WEDNESDAY,THURSDAY,FRIDAY]
 # so that they are immediately available in the system (default is false)
 #
 #org.openhab.inbox:autoApprove=true
+
+################ REST API ###################
+
+# Setting this to true will allow basic authentification for the rest api
```

Comment:
Could you also use the same description as in https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.io.rest.auth/src/main/resources/OH-INF/config/config.xml#L11-L12?